ccf_transaction_processor/Makefile

```diff
@@ -35,7 +39,7 @@ NINJA ?= ninja $(NINJA_OPTIONS)
 SCRIPTDIR ?= $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CCFDSTDIR ?= $(PDO_INSTALL_ROOT)/opt/pdo/ccf
 SRCDIR ?= $(abspath $(SCRIPTDIR)/..)
-KEYDIR := $(CCFDSTDIR)/keys/
+KEYDIR := $(PDO_LEDGER_KEY_ROOT)
 ETCDIR := $(CCFDSTDIR)/etc/
```

Comment:
I am not sure I entirely agree with defaulting KEYDIR here to ${PDO_INSTALL_ROOT}/opt/pdo/etc/keys/ledger rather than $(CCFDSTDIR)/keys/. When CCF is getting installed, keys are a property of CCF rather than PDO, so it made sense to default the key directory to a location within CCF DST directory. Also, given that CCF and PDO never are going to run on the same machine, there is no reason to create ${PDO_INSTALL_ROOT}/opt/pdo/etc/keys/ledger on the machine where CCF is getting installed.

Reply:
the problem is that you're always running ccf on the server where you run the enclave service. this separation is a big pain to maintain when you run them on different servers. its pretty easy to set PDO_LEDGER_KEY_ROOT when you build the CCF TP. That's really the only time this should be anywhere other than the pdo ledger key.

Reply:
> We should at some point pull out CCFDSTDIR out of $(PDO_INSTALL_ROOT)/opt/pdo.

I believe I was trying to get you to do that more cleanly a while back. ;)